application/api/example.1/wait.js

```diff
@@ -0,0 +1,3 @@
+async ({ delay }) => new Promise(resolve => {
```

Comment:
I'd prefer to use the `util.promisify(setTimeout)` defined somewhere as it is more robust and handles more use cases (like new AbortController) (PTAL node-core:`lib/timers/promises.js`)